M3T1/BLL/Models/CartEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿public class CartEntity
```

Comment:
As per requirements, we should have Cart and each cart should have some cart items.